tests/testthat/test-copy-model-from.R

```diff
@@ -61,6 +61,21 @@ test_that("copy_from_model options work [BBR-CMF-002]", {
     stringr::str_extract(orig_mod_str, prob_pattern),
     stringr::str_extract(new_mod_str, prob_pattern)
   )
+
+
+  #Editing YAML to add star status is true from clean model
+  temp_mod_path <- create_temp_model()
+  new_mod <- read_model(temp_mod_path)
+  new_mod <- set_star_status(new_mod, TRUE)
+
+  #Checking that star is written out to the yaml
+  model_yml <- paste0(temp_mod_path, ".yaml") %>% read_yaml() %>% names()
+  expect_equal( model_yml[5],"star")
```

Comment:
I'm not good about this one myself, but `expect_equal` -> `expect_identical` would be better.